n8n/umbrel-app.yml

```diff
@@ -2,12 +2,59 @@ manifestVersion: 1
 id: n8n
 category: automation
 name: n8n
-version: "0.234.1"
+version: "1.4.0"
 releaseNotes: >-
-  This update includes new features, bug fixes, and performance improvements.
+  Bug Fixes
 
+  - core: Add recoveryInProgress flag file
 
-  Full details for changes made since version 0.212.0 can be found at https://github.com/n8n-io/n8n/releases
+  - core: Fix continueOnFail for expression error in Set
+
+  - core: Fix import:workflow command
+
+  - core: Replace throw with warning when deactivating a non-active workflow
+
+  - core: Set up OAuth2 cred test
+
+  - editor: Do not flag dynamic load options issue on expression
+
+  - editor: Ensure community node install button tracks user agreement
+
+  - editor: Fix parsing for single quoted resolvables
+
+  - editor: Fix Remove all fields not removing values in resource mapper
+
+  - editor: Prevent Code node linter from erroring on null parse
+  
+  - Google Sheets Node: Fix short sheet name interpreted as range
+
+  - Google Sheets Trigger Node: Support sheet names with non-latin characters
+
+  - GraphQL Node: Improve error handling
+
+  - Mautic Node: Fix issue with owner not being set correctly
+
+  - Salesforce Node: Fix Account update owner operation
+
+  - Shopify Node: Fix pagination when using options
+
+  - Webhook Node: Backward compatible form-data parsing for non-array fields
+  
+
+  Features
+
+  - core: Add a warning to error workflows that cannot be started due to permission or settings (#6961) (67b88f7)
+
+  - core: Add support for ready hooks, and credentials overwrite endpoint in workers (#6954) (8f8a1de)
+
+  - editor: Show banner for non-production licenses (#6943) (413570c)
+
+  - Remove PostHog event calls (#6915) (270946a)
+  
+  - Send Email Node: Add support for sending text and html email simultaneously (#6978) (3860d41)
+
+
+  Full details for changes made since version 1.4.0 can be found at https://github.com/n8n-io/n8n/releases/tag/n8n%401.4.0
```

Comment:
I added extra newlines here so that the release notes are rendered nicely in the UI like this:
<img width="1285" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/8cf8da04-d18b-4135-939e-ca980b24cabd">

Without a blank newline between textblocks, it ends up looking like this:
<img width="1285" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/9f58b055-7944-49b1-93ff-c15122f9a277">